manifests/profiles/openshift.yaml

```diff
@@ -16,6 +16,11 @@ spec:
        - kube-system
       logLevel: info
       privileged: true
+      providers:
+        multus:
+          globalNetworkAttachmentDefinition:
+            enabled: true
+            namespace: default
     sidecarInjectorWebhook:
       injectedAnnotations:
-        k8s.v1.cni.cncf.io/networks: istio-cni
+        k8s.v1.cni.cncf.io/networks: default/istio-cni
```

Comment:
Don't have enough context - is multus an openshift-only thing ?

Reply:
No, it's just another CNI. It's used to create separate vlans for things like the k8s control plane and the k8s dataplane. You can create more vlans in addition.